python/pyspark/ml/regression.py

```diff
@@ -425,6 +425,10 @@ class DecisionTreeRegressor(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredi
     True
     >>> model.depth == model2.depth
     True
+    >>> isinstance(dt.setVarianceCol("varianceColTest"), DecisionTreeRegressor)
+    True
+    >>> dt.getVarianceCol() == "varianceColTest"
+    True
```

Comment:
This is really just testing set/get of params. I think what @jkbradley was intending is something like L414 above where `model.transform(test1).head().prediction` checks extracting the value for prediction col for one row. You should be able to do something similar here.